java-jacoco/src/main/java/org/sonar/plugins/jacoco/UnitTestAnalyzer.java

```diff
@@ -40,32 +45,21 @@
 import org.sonar.java.JavaClasspath;
 import org.sonar.plugins.java.api.JavaResourceLocator;
 
-import javax.annotation.Nullable;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-
 
-public abstract class AbstractAnalyzer {
+public class UnitTestAnalyzer {
 
   private final ResourcePerspectives perspectives;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ResourcePerspectives"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)